package.json

```diff
@@ -254,6 +254,7 @@
     "@chromatic-com/storybook": "1.2.23",
     "@leather-wallet/prettier-config": "0.0.5",
     "@leather-wallet/rpc": "0.3.0",
+    "@leather-wallet/panda-preset": "0.0.5",
```

Comment:
I manually published the package in this PR - https://github.com/leather-wallet/mono/pull/151 

I did some testing first creating my own packages and wanted to try this properly.